examples/ReorderExample.js

```diff
@@ -65,42 +66,59 @@ class ReorderExample extends React.Component {
       columnOrder.splice(index, 0, event.reorderColumn);
     } else {
       if (fixedColumns.indexOf(event.reorderColumn) !== -1) {
-        columnOrder.splice(fixedColumns.length - 1, 0, event.reorderColumn)
+        columnOrder.splice(fixedColumns.length - 1, 0, event.reorderColumn);
       } else {
         columnOrder.push(event.reorderColumn);
       }
     }
     this.setState({
-      columnOrder: columnOrder
+      columnOrder: columnOrder,
     });
   }
 
-  render() {
-    var {dataList} = this.state;
+  toggleCellsRecycling(enable, columnKey) {
+    this.setState({
+      recycling: {
+        [columnKey]: enable,
+      },
+    });
+  }
 
+  render() {
+    var { dataList, recycling } = this.state;
+    var onColumnReorderEndCallback = this._onColumnReorderEndCallback;
+    const toggleCellsRecycling = this.toggleCellsRecycling;
     return (
       <Table
         rowHeight={30}
         headerHeight={50}
         rowsCount={dataList.getSize()}
-        onColumnReorderEndCallback={this._onColumnReorderEndCallback}
         isColumnReordering={false}
         width={1000}
         height={500}
-        {...this.props}>
+        {...this.props}
+      >
         {this.state.columnOrder.map(function (columnKey, i) {
-          return <Column
-            allowCellsRecycling={true}
-            columnKey={columnKey}
-            key={i}
-            isReorderable={true}
-            header={<DataCell>{columnTitles[columnKey]}</DataCell>}
-            cell={<TextCell data={dataList} />}
-            fixed={fixedColumns.indexOf(columnKey) !== -1}
-            width={columnWidths[columnKey]}
-           />;
+          return (
+            <Column
+              allowCellsRecycling={_.get(recycling, columnKey, true)}
+              columnKey={columnKey}
+              key={i}
+              header={
+                <Plugins.ResizeReorderCell
+                  toggleCellsRecycling={toggleCellsRecycling}
```

Comment:
Users have to manually disable cells recycling

Reply:
Instead of exposing `toggleCellsRecycling` which is an implementation detail, have a callback like `onColumnReorderStart` which can serve the same purpose.

Reply:
`toggleCellsRecycling ` seems to be a meaningful name as per the functionality of method

Reply:
Done